### ISSUE TITLE
Pull request checks seem to be constantly failing

### ISSUE THREAD

**Issue by iocanel** [q-shift/backstage-playground#180]
The checks we have for pull requests are failing. 
We should disable the failing bits and re-work on them.

**Comment by iocanel** [comment]
cc @cmoulliard

**Comment by cmoulliard** [comment]
> We should disable the failing bits and re-work on them.

I agree. Apparently the scenario we are testing don't work anymore during scaffolding step where we create/populate a quarkus application. 

This is certainly related to the fact that the default version we use is not aligned anymore with defaults used on code.quarkus.io where recommended is `3.14` and JDK `17`
```
{
    "id": 2,
    "taskId": "1a09f4af-55ed-48f7-9842-ff1c3cddc6f0",
    "body": {
      "message": "Beginning step Generating the Source Code Component",
      "stepId": "quarkus-app-create",
      "status": "processing"
    },
    "type": "log",
    "createdAt": "2024-09-25T09:07:45.000Z"
  },
  {
    "id": 3,
    "taskId": "1a09f4af-55ed-48f7-9842-ff1c3cddc6f0",
    "body": {
      "message": "AxiosError: Request failed with status code 400\n    at settle (/home/runner/work/backstage-playground/backstage-playground/node_modules/axios/lib/core/settle.js:19:12)\n    at IncomingMessage.handleStreamEnd (/home/runner/work/backstage-playground/backstage-playground/node_modules/axios/lib/adapters/http.js:589:11)\n    at IncomingMessage.emit (node:events:531:35)\n    at endReadableNT (node:internal/streams/readable:1696:12)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)\n    at Axios.request (/home/runner/work/backstage-playground/backstage-playground/node_modules/axios/lib/core/Axios.js:45:41)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Object.handler (/home/runner/work/backstage-playground/backstage-playground/plugins/quarkus-backend/src/scaffolder/actions/quarkus/createApp.ts:143:30)\n    at async NunjucksWorkflowRunner.executeStep (/home/runner/work/backstage-playground/backstage-playground/node_modules/@backstage/plugin-scaffolder-backend/src/scaffolder/tasks/NunjucksWorkflowRunner.ts:380:9)\n    at async NunjucksWorkflowRunner.execute (/home/runner/work/backstage-playground/backstage-playground/node_modules/@backstage/plugin-scaffolder-backend/src/scaffolder/tasks/NunjucksWorkflowRunner.ts:513:9)\n    at async TaskWorker.runOneTask (/home/runner/work/backstage-playground/backstage-playground/node_modules/@backstage/plugin-scaffolder-backend/src/scaffolder/tasks/TaskWorker.ts:176:26)\n    at async run (/home/runner/work/backstage-playground/backstage-playground/node_modules/p-queue/dist/index.js:163:29)",
      "stepId": "quarkus-app-create",
      "status": "failed"
    },
    "type": "log",
    "createdAt": "2024-09-25T09:07:46.000Z"
  },
  {
    "id": 4,
    "taskId": "1a09f4af-55ed-48f7-9842-ff1c3cddc6f0",
    "body": {
      "message": "Run completed with status: failed",
      "error": {
        "name": "AxiosError",
        "message": "Request failed with status code 400"
      }
    },
```

**Comment by cmoulliard** [comment]
Fixed with PR #181
